test/topology_experimental_raft/test_raft_service_levels.py

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+import pytest
+import time
+import asyncio
+import logging
+from test.pylib.util import read_barrier, unique_name, wait_for_cql, wait_for_cql_and_get_hosts
+from cassandra import ConsistencyLevel
+from cassandra.query import SimpleStatement
+
+logger = logging.getLogger(__name__)
+
+@pytest.mark.asyncio
+async def test_service_levels(manager):
```

Comment:
I've talked with @kbr- about the test and he said snapshot transfer is also executed when a new node joins the cluster.
Empirically, the test fails when I do not include service_levels_mutations in snapshot transfer.

Reply:
Yes, when a node joins then the current raft leader sends its most recent snapshot. However, as I wrote in the previous comment, right it might work by accident. Snapshots are created every ~1000 raft commands; it might be that you have accidentally reached that limit and the snapshot being transferred is non-empty.

However, the test should not assume that this is the case - maybe we will change the limit in the future and the test will stop checking anything - it will pass even if snapshot transfer is broken.

Therefore, the test should create a snapshot manually in order to be resilient to such an issue.